platform_buckets_api/api.py

```diff
@@ -204,11 +204,18 @@ async def _resolve_bucket(self, request: Request) -> BaseBucket:
         try:
             bucket = await self.service.get_bucket(id_or_name)
         except NotExistsError:
+            org_name_qv = request.query.get("org_name")
+            project_name_qv = request.query.get("project_name")
             owner_qv = request.query.get("owner")
             user = await _get_untrusted_user(request)
+            if org_name_qv is None and project_name_qv is None:
+                owner_qv = owner_qv or user.name
```

Comment:
say, there's user `a`, and by default they have project `a`. now `a` adds user `b` to their project `a`. `b` adds a bucket to project `a`.
it might be that when `a` tries to get their buckets, they won't see the bucket added by `b`, because we only use `owner` for filtering by default, if any old clients are used.
of course this is an obscure case, just something to think about.